### DISCUSSION TITLE
How to configure jcabi-aspects

### DISCUSSION THREAD

**Discussion by dominsights** [freefair/gradle-plugins#520] [Show and tell]
Gradle 7:

Go to your build.gradle:

Plugins section should contain the freefair compile weaving plugin like this:
```
plugins {
    id "io.freefair.aspectj.post-compile-weaving" version "6.4.3"
}
```

In your dependencies section you should add:

```
dependencies {
    implementation 'org.aspectj:aspectjrt:1.9.9.1'
    implementation 'com.jcabi:jcabi-aspects:0.23.2'

    aspect group: 'com.jcabi', name: 'jcabi-aspects', version: '0.23.2'
}
```

And that's it! I hope this helps someone in the future. The information online is confusing for new gradle users.

**Comment by l3r8yJ** [comment]
@dominsights Hi. For me the method you showed doesn't work, maybe you know other ways to do it?

i'm using it with kotlin project
